.tool-versions

```diff
@@ -0,0 +1,3 @@
+scarb 2.8.2
```

Comment:
@jrcarlos2000 @Nadai2010 are we committing this file?

Reply:
I think it is useful, at least the version of Scarb and starknet-devnet, can commit IMO